pipelines/vars/forklift_luna.yml

```diff
@@ -30,6 +30,9 @@ server_box:
         - "--enable-foreman-plugin-openscap"
         - "--enable-foreman-proxy-plugin-openscap"
 
+        - "--enable-foreman-cli-puppet"
+        - "--enable-foreman-plugin-puppet"
```

Comment:
Do I need to care that these are not available on 2.5 release? Are we running this pipeline there?

Reply:
I did to be on the safe side

Reply:
we only *run* luna for nightly, but there is nothing that forbids a user to run this with `pipeline_version` parameter, so thanks!